### ISSUE TITLE
Bring up SegNet model in TTNN

### ISSUE THREAD

**Issue by dvartaniansTT** [tenstorrent/tt-metal#9695]
Bring up SegNet in TTNN

**Comment by mywoodstock** [comment]
@dvartaniansTT is this still needed? There's the other one: https://github.com/tenstorrent/tt-metal/issues/10754

**Comment by dvartaniansTT** [comment]
yes @mywoodstock ! this one is a different vairant asked by a different customer. but we do have another issue (related to this one) filed with specific unit tests. I will link it here.

**Comment by dvartaniansTT** [comment]
relevant issues:
 https://github.com/tenstorrent/tt-metal/issues/6326
https://github.com/tenstorrent/tt-metal/issues/9694

**Comment by mywoodstock** [comment]
@dvartaniansTT the AI for CNN team are the following:
1. support dilation > 1 in convs
2. implement conv2dTranspose op

Is the model bringup done?

**Comment by mbahnasTT** [comment]
@HariniMohan0102 @saichandax 
can we share with Abhinav on your model bring up branch? is it that one https://github.com/tenstorrent/tt-metal/compare/main...harini/model_net?

**Comment by dvartaniansTT** [comment]
@mbahnasTT I have shared the model implementation with several work arounds with @mywoodstock offline. 

@mywoodstock please note, the details of the parmaters, input, output to the convTranspose2D needed for the customer model can be seen from the torch graph included in the zip file I shared offline with you.
